### PR TITLE
Add the silverwind to proxy.

### PR DESCRIPTION
**Pull request by lsk569937453**
Add the silverwind to proxy.

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [ ] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [ ] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding has working installation instructions.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Review by Hayderhammami — APPROVED** [review]

**Review by Frank220412 — COMMENTED** [review]
B,kzksksjzhdjz

File: README.md [hunk 1]
```diff
@@ -1635,6 +1635,7 @@ _Related: [Web Servers](#web-servers)_
 - [Pomerium](https://www.pomerium.io) - An identity-aware reverse proxy, successor to now obsolete oauth_proxy. It inserts an OAuth step before proxying your request to the backend, so that you can safely expose your self-hosted websites to public Internet. ([Source Code](https://github.com/pomerium/pomerium)) `Apache-2.0` `Go`
 - [Privoxy](https://www.privoxy.org) - Non-caching web proxy with advanced filtering capabilities for enhancing privacy, modifying web page data and HTTP headers, controlling access, and removing ads and other obnoxious Internet junk. `GPL-2.0` `C`
 - [sish](https://github.com/antoniomika/sish) - Open source serveo/ngrok alternative providing HTTP(S)/WS(S)/TCP tunnels to localhost using only SSH. `MIT` `Go`
+- [silverwind](https://github.com/lsk569937453/silverwind) - Open source lightweight and high performance reverse proxy/api gateway.Silverwind has supported AllowDenyList,Authentication,Ratelimiting and so on. ([Demo](https://github.com/lsk569937453/silverwind/wiki)) `Apache-2.0` `Rust`
```
Comment (Rabenherz112) [inline README.md:1638]:
```suggestion
- [silverwind](https://github.com/lsk569937453/silverwind) - Open source lightweight and high performance reverse proxy/api gateway. Silverwind has supported AllowDenyList, Authentication, Ratelimiting and so on. ([Demo](https://github.com/lsk569937453/silverwind/wiki)) `Apache-2.0` `Rust`
```

**Review by Rabenherz112 — CHANGES_REQUESTED** [review]
You missed some spaces.

File: README.md (hunk 1 above)
Comment (Rabenherz112) [reply README.md:1638]: Also, shouldn't it be Rust instead of PHP?

Comment (lsk569937453) [reply README.md:1638]: Yeah.It is Rust.

**Review by Hayderhammami — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1629,6 +1629,7 @@ _Related: [Web Servers](#web-servers)_
 - [Pomerium](https://www.pomerium.io) - An identity-aware reverse proxy, successor to now obsolete oauth_proxy. It inserts an OAuth step before proxying your request to the backend, so that you can safely expose your self-hosted websites to public Internet. ([Source Code](https://github.com/pomerium/pomerium)) `Apache-2.0` `Go`
 - [Privoxy](https://www.privoxy.org) - Non-caching web proxy with advanced filtering capabilities for enhancing privacy, modifying web page data and HTTP headers, controlling access, and removing ads and other obnoxious Internet junk. `GPL-2.0` `C`
 - [sish](https://github.com/antoniomika/sish) - Open source serveo/ngrok alternative providing HTTP(S)/WS(S)/TCP tunnels to localhost using only SSH. `MIT` `Go`
+- [silverwind](https://github.com/lsk569937453/silverwind) - Open source lightweight and high performance reverse proxy/api gateway. Silverwind has supported AllowDenyList, Authentication, Ratelimiting and so on. ([Demo](https://github.com/lsk569937453/silverwind/wiki)) `Apache-2.0` `Rust`
```
Comment (Hayderhammami) [inline README.md:1632]: ebf824c112d20c78d6d35ef2cd5dfe1b14ac95b6

**Pull request closed without merging**